pints/_mcmc/_monomial_gamma_hmc.py

```diff
@@ -0,0 +1,507 @@
+#
+# Monomial gamma HMC MCMC method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+from scipy import integrate
+from scipy import interpolate
+
+
+class MonomialGammaHMCMCMC(pints.SingleChainMCMC):
+    """
+    *Extends:* :class:`SingleChainMCMC`
```

Comment:
Could you move this line to the bottom of the docstring please?

Reply:
Done